README.md

```diff
@@ -22,7 +22,11 @@ elm-app start
 navigate to http://localhost:3000
 
 ## Test
-In the root director run elm test
+Install the latest version of elm-test for elm 0.19
 ```
-elm test
+sudo npm install -g elm-test@0.19.0-beta4
```

Comment:
Why not just add this to the test-dependencies.  Will those get installed when you npm install?  That way we could just run npm install and make test

Reply:
Good idea

Reply:
I've added the correct version of elm-test to a package.json file for now